source/adios2/toolkit/transport/file/FileFStream.cpp

```diff
@@ -354,6 +354,24 @@ void FileFStream::Seek(const size_t start)
     }
 }
 
+#if __cplusplus >= 201703L
+#include <filesystem>
```

Comment:
I am getting a huge lot of errors when compiling this line using GCC 9.3.0

In file included from /usr/include/c++/9/filesystem:36,
                 from /home/jason/d/a2/source/adios2/toolkit/transport/file/FileFStream.cpp:358:
/usr/include/c++/9/bits/fs_fwd.h: In function ‘constexpr adios2::transport::std::filesystem::copy_options adios2::transport::std::filesystem::operator&(adios2::transport::std::filesystem::copy_options, adios2::transport::std::filesystem::copy_options)’:
/usr/include/c++/9/bits/fs_fwd.h:91:35: error: ‘underlying_type’ in namespace ‘adios2::transport::std’ does not name a template type
   91 |     using __utype = typename std::underlying_type<copy_options>::type;
      |                                   ^~~~~~~~~~~~~~~
/usr/include/c++/9/bits/fs_fwd.h:93:14: error: ‘__utype’ does not name a type
   93 |  static_cast<__utype>(__x) & static_cast<__utype>(__y));
      |              ^~~~~~~